src/lib/falcor/routes/articles/by-slug.js

```diff
@@ -137,40 +56,6 @@ export default [
         });
       }),
   },
-  {
-    // Add authors to an article
-    route: "articles['bySlug'][{keys:slugs}]['authors']['updateAuthors']",
-    call: (callPath, args) => {
-      // the falcor.model.call only takes a path not a pathset
-      // so it is not possible to call this function for more
-      // than 1 article at a time, therefore we know keys:slugs is length 1
-      if (callPath.slugs > 1) {
-        throw new Error(
-          'updateAuthors falcor function was called illegally with more than 1 article slug',
-        );
-      }
-      return new Promise(resolve => {
-        const articleId = args[0];
-        const newAuthors = args[1];
-        const articleSlug = callPath.slugs[0];
-        updateAuthors(articleId, newAuthors).then(data => {
-          const results = [];
-          // Invalidate all the old data
-          results.push({
-            path: ['articles', 'bySlug', articleSlug, 'staff'],
```

Comment:
@emilgoldsmith These are the remaining staff that I want to make sure were a mistake. They definitely were causing bugs (the path to get this info is 'authors' but this is invalidating 'staff'), and these were the reason I changed everything to Staff from Authors, but I want to make sure the inconsistency wasn't for some reason I have missed.